### PR TITLE
Long running test fixes and better error reporting

### PR DESCRIPTION
**Pull request by brianjohnson5972**
## Change Description

Fixed bad path to integration_test.wasm and integration_test.abi.

## Consensus Changes

N/A

## API Changes

N/A

## Documentation Additions

N/A

### CODE REVIEW

File: tests/nodeos_forked_chain_test.py [hunk 1]
```diff
@@ -348,8 +348,12 @@ def getMinHeadAndLib(prodNodes):
         if blockProducer0!=blockProducer1:
             Utils.errorExit("Groups reported different block producers for block number %d. %s != %s." % (blockNum,blockProducer0,blockProducer1))
 
+    #verify that the non producing node is not alive (and populate the producer nodes with current getInfo data to report if
+    #an error occurs)
+    if nonProdNode.verifyAlive():
+        Utils.errorExit("Expected the non-producing node to have shutdown.")
 
-    # ***   Analyze the producers leading up to the block after killing the non-producing node   ***
+    Print("Analyze the producers leading up to the block after killing the non-producing node")
```
Comment (jgiszczak) [inline tests/nodeos_forked_chain_test.py]: 'Analyzing' rather than 'Analyze'

File: tests/nodeos_forked_chain_test.py [hunk 2]
```diff
@@ -358,15 +362,11 @@ def getMinHeadAndLib(prodNodes):
...
-    #verify that the non producing node is not alive (and populate the producer nodes with current getInfo data to report if
-    #an error occurs)
-    if nonProdNode.verifyAlive():
-        Utils.errorExit("Expected the non-producing node to have shutdown.")
     for prodNode in prodNodes:
         prodNode.getInfo()
 
 
-    # ***   Track the blocks from the divergence till there are 10*12 blocks on one chain and 10*12+1 on the other   ***
+    Print("Track the blocks from the divergence till there are 10*12 blocks on one chain and 10*12+1 on the other")
```
Comment (jgiszczak) [inline tests/nodeos_forked_chain_test.py]: 'Tracking'

File: tests/nodeos_forked_chain_test.py [hunk 3]
```diff
@@ -377,7 +377,7 @@ def getMinHeadAndLib(prodNodes):
         blockProducers1.append({"blockNum":blockNum, "prod":blockProducer1})
 
 
-    # ***   Analyze the producers from the divergence to the lastBlockNum and verify they stay diverged   ***
+    Print("Analyze the producers from the divergence to the lastBlockNum and verify they stay diverged")
```
Comment (jgiszczak) [inline tests/nodeos_forked_chain_test.py]: 'Analyzing'

File: tests/nodeos_forked_chain_test.py [hunk 4]
```diff
@@ -386,13 +386,13 @@ def getMinHeadAndLib(prodNodes):
     blockProducers1=[]
 
 
-    # ***   Relaunch the non-producing bridge node to connect the producing nodes again   ***
+    Print("Relaunch the non-producing bridge node to connect the producing nodes again")
```
Comment (jgiszczak) [inline tests/nodeos_forked_chain_test.py]: 'Relaunching'

File: tests/nodeos_forked_chain_test.py [hunk 5]
```diff
@@ -386,13 +386,13 @@ def getMinHeadAndLib(prodNodes):
...
 
     if not nonProdNode.relaunch(nonProdNode.nodeNum, None):
         errorExit("Failure - (non-production) node %d should have restarted" % (nonProdNode.nodeNum))
 
 
-    # ***   Identify the producers from the saved LIB to the current highest head   ***
+    Print("Identify the producers from the saved LIB to the current highest head")
```
Comment (jgiszczak) [inline tests/nodeos_forked_chain_test.py]: 'Identifying'

File: tests/nodeos_forked_chain_test.py [hunk 6]
```diff
@@ -407,7 +407,7 @@ def getMinHeadAndLib(prodNodes):
         blockProducers1.append({"blockNum":blockNum, "prod":blockProducer1})
 
 
-    # ***   Analyze the producers from the saved LIB to the current highest head and verify they match now   ***
+    Print("Analyze the producers from the saved LIB to the current highest head and verify they match now")
```
Comment (jgiszczak) [inline tests/nodeos_forked_chain_test.py]: 'Analyzing'

File: tests/nodeos_forked_chain_test.py (hunk 1 above)
Comment (jgiszczak) [reply tests/nodeos_forked_chain_test.py]: Many changed, but this one and the next didn't.

Comment (brianjohnson5972) [reply tests/nodeos_forked_chain_test.py]: Fixed now.

**Pull request merged**